integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/message/processor/test/DeactivatedCappMPBehaviourOnRestartTestCase.java

```diff
@@ -80,15 +68,13 @@ public void testDeactivatedMPUponServerRestart() throws Exception {
         axis2Server.start();
 
         // Restart the ESB Server after the MP is deactivated.
-        serverConfigurationManager.restartGracefully();
+        CarbonServerExtension.restartServer();
 
         super.init();
-        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
+        CarbonLogReader carbonLogReader = new CarbonLogReader();
 
-        Assert.assertFalse(Utils.checkForLog(logViewerClient, EXPECTED_ERROR_MESSAGE, 10),
+        Assert.assertFalse(Utils.logExists(carbonLogReader, EXPECTED_ERROR_MESSAGE, 10),
```

Comment:
Since the log reader is not started the , the message will not be there anyway .

Reply:
Fixed.

Reply:
Fixed.